client/allocrunner/taskrunner/template/template.go

```diff
@@ -574,6 +574,13 @@ func parseTemplateConfigs(config *TaskTemplateManagerConfig) (map[ctconf.Templat
 		ct.LeftDelim = &tmpl.LeftDelim
 		ct.RightDelim = &tmpl.RightDelim
 
+		// By default we pass a blacklist of functions to prevent
+		// task operators from bypassing client-task isolation.
+		// This protection can be disabled by the client config.
+		if !config.ClientConfig.EnableInsecureTemplateFunctions {
```

Comment:
this line and the change to the documentation suggest that this is supposed to be in the client config, but the change to the parsing show it in the top-level config. as such, right now it's not possible to change this to a non-default value of `true`.

Reply:
Ah yes, thanks for catching that.